ravenframework/Models/EnsembleModel.py

```diff
@@ -318,17 +318,20 @@ def initialize(self,runInfo,inputs,initDict=None):
         outputMatch.extend(match if match is not None else [])
       outputMatch = list(set(outputMatch))
       modelsToOutputModels[modelIn] = outputMatch
+    executionList, modelsGraph, _ = evaluateModelsOrder(modelsToOutputModels, acceptLoop=True, reverse=False, initialStartingModels=self.initialStartModels)
     # construct the ensemble model directed graph
-    self.ensembleModelGraph = graphStructure.graphObject(modelsToOutputModels)
+    #self.ensembleModelGraph = graphStructure.graphObject(modelsToOutputModels)
+    self.ensembleModelGraph = modelsGraph #graphStructure.graphObject(modelsToOutputModels)
     # make some checks
     if not self.ensembleModelGraph.isConnectedNet():
       isolatedModels = self.ensembleModelGraph.findIsolatedVertices()
       self.raiseAnError(IOError, "Some models are not connected. Possible candidates are: "+' '.join(isolatedModels))
     # get all paths
-    allPath = self.ensembleModelGraph.findAllUniquePaths(self.initialStartModels)
+    #allPath = self.ensembleModelGraph.findAllUniquePaths(self.initialStartModels)
     ###################################################
     # to be removed once executionList can be handled #
-    self.orderList = self.ensembleModelGraph.createSingleListOfVertices(allPath)
+    #self.ensembleModelGraph.createSingleListOfVertices(allPath)
```

Comment:
Do you want to keep the commented lines?

Reply:
removed